app/backend/src/app.py

```diff
@@ -60,7 +61,12 @@
 
 logging.getLogger("couchers.jobs.worker").setLevel(logging.INFO)
 
+if config.config["SENTRY_ENABLED"]:
+    """Sends exception tracebacks to Sentry, a cloud service for collecting exceptions"""
+    sentry_sdk.init(config.config["SENTRY_URL"], traces_sample_rate=0.0)
+
 start_http_server(port=8000, registry=main_process_registry)
+"""Starts a prometheus metrics endpoint on port 8000"""
```

Comment:
Ditto, but also you probably want this above the line?

Reply:
was my attempt to follow the google docstring format, will change back to regular comments